cmd/get.go

```diff
@@ -22,6 +22,7 @@ func NewGetCmd(global *globalFlags) *cobra.Command {
 	cc := &cobra.Command{
 		Use:   "get",
 		Short: "Get cached secret value",
+		Long:  "Retrieves the cached secret value which has been either set by using the provided `set` command, or by setting environment variables. The environment variables have the following format: `CRYPTA_SET_<KEY>`. The value given for `<KEY>` will be used as the name with which the value of the environment variable will be added to the secret store.",
```

Comment:
@codetent Let me know if this is OK for you, or whether you prefer to also add this info to the README.md page?